src/module/rules/SoakRules.ts

```diff
@@ -29,6 +29,21 @@ export class SoakRules {
             return Helpers.reduceDamageByHits(damageData, damageData.value, 'SR5.VehicleStunImmunity');
         }
 
+        let armorData = actor.getArmor(damageData)
+
+        // @ts-expect-error
+        if(armorData.hardened) {
+            let ap = armorData.mod.filter(mod => mod.name === "SR5.AP")[0].value
```

Comment:
You should use `damageData.ap.value` here, but you also shouldn't need it.  `SR5Actor:getArmor` considers AP and calculates its value accordingly, so I think this could be simplified to just `damageData.value < armorData.value`

Reply:
In my tests getArmor provided the unmodified armor. getModifiedArmor was the same, that why I filter the mods, but I can look again if I missed something